assets/storageclass.yaml

```diff
@@ -6,4 +6,4 @@ provisioner: csi.vsphere.vmware.com
 parameters:
   datastoreurl: "${DATASTORE_URL}"
 volumeBindingMode: WaitForFirstConsumer
-allowVolumeExpansion: true
+allowVolumeExpansion: false
```

Comment:
It could be interesting to add a comment stating why this is currently disabled with a link to the ticket/issue that needs to be fixed.

Reply:
Also, we need to run `make update` to update the bindata file and include that in this PR.

Reply:
may be we need a verify job to catch failure to run `make update`  somewhere

Reply:
actually resize only support block volume, not vSAN, I think.

Reply:
hmm, so I think at some point this document was changed. But I could confirm with vmware engineers again. In any case - volume expansion is still beta and hence it makes sense for us to keep it disabled until we could test and validate it.